sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -389,11 +381,8 @@ export class BatchingReceiver extends MessageReceiver {
         this._receiver!.addCredit(maxMessageCount);
         let msg: string = "[%s] Setting the wait timer for %d seconds for receiver '%s'.";
         if (reuse) msg += " Receiver link already present, hence reusing it.";
-        log.batching(msg, this._context.namespace.connectionId, maxWaitTimeInSeconds, this.name);
-        totalWaitTimer = setTimeout(
-          actionAfterWaitTimeout,
-          (maxWaitTimeInSeconds as number) * 1000
-        );
+        log.batching(msg, this._context.namespace.connectionId, maxWaitTimeInMs / 1000, this.name);
```

Comment:
They can choose a non-whole increment of a second so this should probably log the ms number, no division.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8169/commits/1313eb66765b5cc1fe2db2ce12c335d842f79db4